frontend/src/components/editCard/ModifyEdit.tsx

```diff
@@ -291,7 +298,10 @@ type ScenePerformance = {
 
 interface URL {
```

Comment:
This has a duplicate in `URLChangeRow`. Maybe better to remove this one and import from `URLChangeRow` instead.